marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/artifacts/matching.sjs

```diff
@@ -20,10 +20,10 @@ const DataHubSingleton = require('/data-hub/5/datahub-singleton.sjs');
 // define constants for caching expensive operations
 const dataHub = DataHubSingleton.instance();
 
-const collections = ['http://marklogic.com/data-hub/matching-artifact'];
+const collections = ['http://marklogic.com/data-hub/steps/matching', 'http://marklogic.com/data-hub/steps'];
 const databases = [dataHub.config.STAGINGDATABASE, dataHub.config.FINALDATABASE];
-const permissions = [xdmp.permission(dataHub.consts.DATA_HUB_MATCHING_WRITE_ROLE, 'update'), xdmp.permission(dataHub.consts.DATA_HUB_MATCHING_READ_ROLE, 'read')];
-const requiredProperties = ['name', 'targetEntityType', 'selectedSource'];
+const permissions = [xdmp.permission(dataHub.consts.DATA_HUB_FLOW_WRITE_ROLE, 'update'), xdmp.permission(dataHub.consts.DATA_HUB_FLOW_READ_ROLE, 'read')];
+const requiredProperties = ['name'];
```

Comment:
@ryanjdew There's currently validation in here, and I agree with dropping targetEntityType and selectedSource. But I'm thinking we shouldn't do any validation at all, as I don't think there's a requirement that matching steps currently have a "name". I think we can tackle validation in 5.4.

Reply:
Removed validation